ballerina/Ballerina.toml

```diff
@@ -2,12 +2,12 @@
 distribution = "2201.8.0"
 org = "ballerinax"
 name = "twilio"
-version = "4.0.1"
```

Comment:
This should be a minor version bump. I assume `4.1.0` is the appropriate version. Update this in the `gradle.properties` as well.

Reply:
Updated!